components/calendly_v2/actions/get-event/get-event.mjs

```diff
@@ -1,11 +1,22 @@
-// legacy_hash_id: a_poiwMj
+////https://developer.calendly.com/api-docs/e2f95ebd44914-get-event
```

Comment:
@malexanderlim check out how / where other registry components include links to the docs (linked in the component description)